CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## securesystemslib v0.32.0
```

Comment:
Should we make this 1.0.0 already?

I submitted a host of PRs yesterday to remove all legacy code (#731). If we merge those, I think we are ready.

Reply:
Before we tag 1.0.0 we should remove disclaimers like this:

https://github.com/secure-systems-lab/securesystemslib/blame/main/docs/signer.rst#L62-L67 

(there may be others)